skrf/network.py

```diff
@@ -400,6 +400,7 @@ def __init__(self, file=None, name=None, comments=None, f_unit=None, s_def=S_DEF
         self.deembed = None
         self.noise = None
         self.noise_freq = None
+        self._z0: npy.ndarray = npy.array(50, dtype=complex)
```

Comment:
```suggestion
        self._z0 = npy.array(50, dtype=complex)
```

Reply:
will this typeing  allow them to set a number  for z0?